src/python/datastore/corpus_tagging.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for managing corpus tags."""
+
+from datastore import data_types
+
+
+def get_fuzz_target_tag(fully_qualified_fuzz_target_name):
+  """Get all the tag of a given fuzz target."""
+  query = data_types.CorpusTag().query()
+  query = query.filter(data_types.CorpusTag.fully_qualified_fuzz_target_name ==
+                       fully_qualified_fuzz_target_name)
+  return list(query)
```

Comment:
don't we need `query.fetch()` here?

also, I think @oliverchang mentioned that `list` isn't really needed, at least here. The caller can puts the result in the list, but otherwise the generator returned by `query.fetch()` should be sufficient

Reply:
Done